gammagl/models/mgnni.py

```diff
@@ -0,0 +1,99 @@
+import numpy as np
+import tensorlayerx as tlx
+import tensorlayerx.nn as nn
+from gammagl.layers.conv import MGNNI_m_iter
+
+
+class MGNNI_m_MLP(tlx.nn.Module):
+    def __init__(self, m, m_y, nhid, ks, threshold, max_iter, gamma, fp_layer='MGNNI_m_att', dropout=0.5,
+                 batch_norm=False):
+        super(MGNNI_m_MLP, self).__init__()
+        self.fc1 = tlx.layers.Linear(out_features=nhid,
+                                     in_features=m,
+                                     # W_init='xavier_uniform',
+                                     b_init=None)
+        self.fc2 = tlx.layers.Linear(out_features=nhid,
+                                     in_features=nhid,
+                                     # W_init='xavier_uniform',
+                                     )
+
+        self.dropout = dropout
+        self.MGNNI_layer = eval(fp_layer)(nhid, m_y, ks, threshold, max_iter, gamma, dropout=self.dropout,
+                                          batch_norm=batch_norm)
+
+    def forward(self, X, edge_index, edge_weight=None, num_nodes=None):
+        # print('I am coming!')
+        X = nn.Dropout(p=self.dropout)(tlx.ops.transpose(X))
+        X = nn.ReLU()(self.fc1(X))
+        X = nn.Dropout(p=self.dropout)(X)
+        X = self.fc2(X)
+        output = self.MGNNI_layer(tlx.ops.transpose(X), edge_index, edge_weight, num_nodes)
+
+        return output
+
+
+class MGNNI_m_att(nn.Module):
```

Comment:
Remember to give a description with docstring refer to model ``gcn``.

Reply:
Thank you for the feedback. I apologize for the oversight. I have added the relevant docstring.